tegra/tegra.c

```diff
@@ -206,7 +206,7 @@ int drm_tegra_bo_unref(struct drm_tegra_bo *bo)
 		return -EINVAL;
 
 	if (!atomic_dec_and_test(&bo->ref))
-		return 0;
+		return atomic_read(&bo->ref);
```

Comment:
Yeah, but the CRT cares about NDEBUG for disabling assert, not DEBUG. So yeah, seems libdrm left some stuff to be desired WRT debuggability vs performance.